kong/db/strategies/postgres/connector.lua

```diff
@@ -314,55 +322,94 @@ function _mt:init()
 end
 
 
-function _mt:init_worker(strategies)
-  if ngx.worker.id() == 0 then
+local function cleanup_expired_rows_in_table(config, table_name)
+  -- Create new connection on each table to avoid reusing a connection that might be already timed out
+  local connector = connect(config)
+
+  local cleanup_start_time = now_updated()
+  local ttl_escaped = connector:escape_identifier("ttl")
+  local expired_at_escaped = connector:escape_identifier("expire_at")
+  local column_name = table_name == "cluster_events" and expired_at_escaped
+                                                      or ttl_escaped
+
+  local cleanup_statement = concat {
+    "DELETE FROM ",
+    connector:escape_identifier(table_name),
+    " WHERE ctid in (SELECT ctid FROM ",
+    connector:escape_identifier(table_name),
+    " WHERE ",
+    column_name,
+    " < TO_TIMESTAMP(%s) AT TIME ZONE 'UTC' LIMIT ",
```

Comment:
Would it be helpful if we force db to use index here? I think it might not be better, but at least it wouldn't be worse than seq scan when using index.

Reply:
I did a google and found no way of using force index in Postgres... it seems that this is not doable
`SELECT`ing a batch of rows here should be fine since we already have `ttl` index for these table, so let's just leave the job for query engine to decide whether it should use index